packages/apps/plugins/plugin-table/src/components/ObjectTable/hooks/useTables.ts

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { useMemo } from 'react';
+
+import { TableType } from '@braneframe/types';
+import { Filter } from '@dxos/echo-schema';
+import { type Space, useQuery } from '@dxos/react-client/echo';
+
+export const useTables = (space?: Space) => {
+  const tableFilter = useMemo(() => Filter.schema(TableType), []);
```

Comment:
is this still needed? does not memoizing this result in a superfluous updates? if so, we should address this at the react client level because everyone would end up needing to do this which isn't ideal

Reply:
You're right. I removed this from the objects query but forgot this one 👍.